jetty-fcgi/fcgi-server/src/test/java/org/eclipse/jetty/fcgi/server/HttpClientTest.java

```diff
@@ -437,7 +435,7 @@ public void handle(String target, org.eclipse.jetty.server.Request baseRequest,
         assertThrows(TimeoutException.class, () ->
             client.newRequest(host, port)
                 .scheme(scheme)
-                .idleTimeout(idleTimeout, TimeUnit.MILLISECONDS)
+                .idleTimeout(idleTimeout * 90 / 100, TimeUnit.MILLISECONDS)
```

Comment:
Not in this case. Since this test is about triggering an idle timeout, as long as the server waits more than what the client's idle timeout, this test does its job. It does not matter if the response was too late because of a GC pause or because of the sleep as long as we can guarantee that the server will take more time than the client is willing to wait. So if we can make sure the server will wait at least 100ms more, that's good enough.

But maybe I should make the server wait `idleTimeout * 110 / 100` and make the client use `idleTimeout = 1000` as its timeout to improve clarity a bit.